src/pages/quiz.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import MultipleChoice from "../components/pages/quiz/MultipleChoice";
+import NumericalAnswer from "components/pages/quiz/NumericalAnswer";
+import "./quiz.css";
+
+const MCQ = [
+  {
+    questionTitle: "Largest Ocean in the world?",
+    answerChoices: ["Pacific", "Indian", "Atlantic", "Arctic"],
+    correctAnswer: "Pacific",
+  },
+  {
+    questionTitle: "7th largest country?",
+    answerChoices: ["India", "China", "USA"],
+    correctAnswer: "India",
+  },
+];
+
+const NAQ = [
+  {
+    questionTitle: "How many hours do you sleep per day?",
+    correctAnswer: "8",
+  },
+];
+
+export default function Quiz() {
+  return (
+    <div className="container">
```

Comment:
Regarding answers, I think radio choices stacked vertically would be better. In addition, I think you should use the Bulma form controls for input related things. There's lots of quiz apps for inspiration, such as:

![image](https://user-images.githubusercontent.com/30810402/93531882-f182e580-f90d-11ea-8984-cc42a94408f2.png)

or something simpler:

![image](https://user-images.githubusercontent.com/30810402/93531916-fe9fd480-f90d-11ea-89db-017b29cf74b6.png)